src/test/java/LoginTest.java

```diff
@@ -0,0 +1,42 @@
+import laba.pages.LoginPage;
+import laba.pages.ProductsPage;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import com.zebrunner.carina.utils.R;
+
+public class LoginTest extends BaseTest {
+
+    @DataProvider(name = "loginData")
+    public Object[][] loginData() {
+        return new Object[][] {
+                {"user.standard", "password.valid", true, ""},
+                {"user.locked", "password.valid", false, "Sorry, this user has been locked out."},
+                {"user.problem", "password.valid", true, ""},
+                {"", "password.valid", false, "Username is required"},
+                {"user.standard", "", false, "Password is required"},
+                {"user.invalid", "password.invalid", false, "Username and password do not match any user in this service."}
+        };
```

Comment:
Define User objects as public static final constants , 
    public static final User STANDARD_USER = new User(R.TESTDATA.get("standard_user_name"), R.TESTDATA.get("standard_user_password"));
and clean data provider
@DataProvider(name = "loginData")
public Object[][] loginData() {
    return new Object[][] {
        {STANDARD_USER, true, ""},
        {LOCKED_USER, false, "Sorry, this user has been locked out."},